app/lib/actions/katello/upstream_subscriptions/remove_entitlements.rb

```diff
@@ -13,7 +13,7 @@ def plan(pool_ids = [])
             ids.each do |pid|
               pool = ::Katello::Pool.find(pid)
 
-              fail("Provided pool with id #{pid} has no upstream entitlement") if pool.upstream_entitlement_id.nil?
+              fail _("Provided pool with id %s has no upstream entitlement" % pid) if pool.upstream_entitlement_id.nil?
```

Comment:
Ok, I was going to be fine with one non-related change, but this is pushing it. I'd rather you add a PR that refs the remove subs issue.

Reply:
No problem - but should it be 'refs' for an issue which is closed - or a new issue altogether?

Reply:
Actually, I think I'll defend the change being included here - it's unit tested and very much related in the sense that I am making the messaging consistent across the two actions

Reply:
It's fine to refs a recently closed issue. It only becomes a problem when the issue has started to be included in a release. Since this stuff is all 3.7.0, we should be fine.

Reply:
I still think you should put it in a separate PR so the remove subs issue points to all the proper changes.